.github/workflows/discord_discussions.yml

```diff
@@ -0,0 +1,52 @@
+name: Discord Discussions
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - labeled
+      - closed
```

Comment:
Why are we doing it for things like edited etc?

Reply:
Reopened/closed for thread state change
Opened/Labeled for initialization
Edited for title changes

Reply:
The job only runs if the label is present